pkgs/by-name/di/dioxionary/package.nix

```
@@ -0,0 +1,58 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  openssl,
+  installShellFiles,
+  nix-update-script,
+  versionCheckHook,
+}:
+rustPlatform.buildRustPackage rec {
```

using finalAttrs instead of rec is preferred
```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```

---

Preferred but not blocking. It also increases eval times slightly, so. You can't go wrong either way, though.

---

It does take a bit more time to eval, but I would prefer if we just use it for good overridability's sake.

---

I agree.

---

i will change to use `finalAttrs`. thanks.